### ISSUE TITLE
Product Rating Filter not keyboard accessible

### ISSUE THREAD

**Issue by ravenokko** [shopware/platform#12942]
### Shopware Version

6.7.3.0

### Affected area / extension

Platform(Default)

### Actual behaviour

When the filter by product rating is active, it is not possible to tab through all the options.

https://github.com/user-attachments/assets/748c7df4-66e5-4f0b-a8c1-679e29390ca1

### Expected behaviour

Dropdown options should be accessible with a mouse, keyboard and screenreader.

### How to reproduce

Activate product rating filter and try to access all the options via keyboard.

**Comment by vintagesucks** [comment]
@ravenokko

The rating filter is implemented as a radio group, and keyboard users should (as far as I know[^1]) use the arrow keys to move between options within the group, since tabbing will move focus into and out of the group, not between individual options.

Arrow key navigation works as expected for the rating filter in my testing:

https://github.com/user-attachments/assets/b6920ee3-66c6-49f4-9fec-1fd01d938260

[^1]: https://www.w3.org/WAI/ARIA/apg/patterns/radio/
